ci/terraform/account-management/api-gateway.tf

```diff
@@ -36,35 +36,13 @@ resource "aws_iam_role_policy" "invocation_policy" {
 EOF
 }
 
-resource "aws_cloudwatch_log_subscription_filter" "authorizer_log_subscription" {
```

Comment:
did you mean to delete these?

Reply:
I've moved them (in the same commit). Makes more sense for them to be in the authoriser.tf